### PR TITLE
vote: `binshi-bing` as pd reviewer

### PR DESCRIPTION
**Pull request by rleungx**

### CODE REVIEW

File: votes/0192-binshi-bing-as-pd-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# A Vote for binshi-bing as TiKV Reviewer
+
+## Proposal
+
+[@binshi-bing](https://github.com/binshi-bing) has continuously contributed to `tikv/pd` on different components, including but not limited to storage, raftstore and related components. The main contributions are listed in the following:
```
Comment (JmPotato) [inline votes/0192-binshi-bing-as-pd-reviewer.md:5]: The description seems that not that matched.

**Review by lhy1024 — APPROVED** [review]

File: votes/0192-binshi-bing-as-pd-reviewer.md (hunk 1 above)
Comment (rleungx) [reply votes/0192-binshi-bing-as-pd-reviewer.md]: fixed

**Review by JmPotato — APPROVED** [review]

**Review by nolouch — APPROVED** [review]
lgtm. thanks @binshi-bing contributions.

**Comment by rleungx** [comment]
@binshi-bing congrats!

**Comment by binshi-bing** [comment]
> @binshi-bing congrats!

Thank you very much for driving this!

**Pull request merged**